### PR TITLE
Remove unique producer merge branch

### PR DESCRIPTION
**Pull request by daniel-davidd**

### CODE REVIEW

File: db/db.go [hunk 1]
```diff
@@ -314,19 +320,13 @@ func createTables(MetadataDbClient MetadataStorage) error {
 		connection_id VARCHAR NOT NULL,
 		consumers_group VARCHAR NOT NULL,
 		max_ack_time_ms SERIAL NOT NULL,
-		created_by INTEGER,
-		created_by_username VARCHAR NOT NULL,
 		is_active BOOL NOT NULL DEFAULT true,
-		created_at TIMESTAMPTZ NOT NULL,
-		is_deleted BOOL NOT NULL DEFAULT false,
+		updated_at TIMESTAMPTZ NOT NULL,
```
Comment (shay23b) [inline db/db.go:324]: move to end of table, when upgraded this column will be created at the end from the alter table and can cause inconsistencies
also change the struct

File: server/memphis_handlers_consumers.go [hunk 2]
```diff
@@ -75,12 +76,12 @@ func GetConsumerGroupMembers(cgName string, station models.Station) ([]models.Cg
 	return dedupedConsumers, nil
 }
 
-func (s *Server) createConsumerDirectV0(c *client, reply string, ccr createConsumerRequestV0, requestVersion int) {
-	err := s.createConsumerDirectCommon(c, ccr.Name, ccr.StationName, ccr.ConsumerGroup, ccr.ConsumerType, ccr.ConnectionId, ccr.MaxAckTimeMillis, ccr.MaxMsgDeliveries, requestVersion, 1, -1)
-	respondWithErr(serv.MemphisGlobalAccountString(), s, reply, err)
+func (s *Server) createConsumerDirectV0(c *client, reply, tenantName string, ccr createConsumerRequestV0, requestVersion int) {
+	err := s.createConsumerDirectCommon(c, ccr.Name, ccr.StationName, ccr.ConsumerGroup, ccr.ConsumerType, ccr.ConnectionId, tenantName, ccr.Username, ccr.MaxAckTimeMillis, ccr.MaxMsgDeliveries, requestVersion, 1, -1)
+	respondWithErr(MEMPHIS_GLOBAL_ACCOUNT, s, reply, err)
```
Comment (shay23b) [inline server/memphis_handlers_consumers.go:81]: change MEMPHIS_GLOBAL_ACCOUNT to serv.MemphisGlobalAccountString() in all relevant places

File: server/memphis_handlers_consumers.go [hunk 3]
```diff
@@ -292,8 +281,8 @@ func (s *Server) createConsumerDirect(c *client, reply string, msg []byte) {
 		return
 	}
 
-	err = s.createConsumerDirectCommon(c, ccr.Name, ccr.StationName, ccr.ConsumerGroup, ccr.ConsumerType, ccr.ConnectionId, ccr.MaxAckTimeMillis, ccr.MaxMsgDeliveries, 1, ccr.StartConsumeFromSequence, ccr.LastMessages)
-	respondWithErr(serv.MemphisGlobalAccountString(), s, reply, err)
+	err = s.createConsumerDirectCommon(c, ccr.Name, ccr.StationName, ccr.ConsumerGroup, ccr.ConsumerType, ccr.ConnectionId, tenantName, ccr.Username, ccr.MaxAckTimeMillis, ccr.MaxMsgDeliveries, 1, ccr.StartConsumeFromSequence, ccr.LastMessages)
+	respondWithErr(MEMPHIS_GLOBAL_ACCOUNT, s, reply, err)
```
Comment (shay23b) [inline server/memphis_handlers_consumers.go:285]: change MEMPHIS_GLOBAL_ACCOUNT to serv.MemphisGlobalAccountString() in all relevant places

File: server/memphis_handlers_consumers.go [hunk 4]
```diff
@@ -576,158 +574,91 @@ func (s *Server) destroyConsumerDirect(c *client, reply string, msg []byte) {
...
-			analytics.SendEvent(user.TenantName, username, analyticsParams, "user-remove-consumer-sdk")
-		}
-	}
-
-	respondWithErr(MEMPHIS_GLOBAL_ACCOUNT, s, reply, nil)
+	s.destroyCGFromNats(c, reply, dcr.Username, tenantName, stationName, consumer, station)
 }
 
-func (s *Server) destroyConsumerDirectV0(c *client, reply string, dcr destroyConsumerRequestV0) {
+func (s *Server) destroyConsumerDirectV0(c *client, reply string, dcr destroyConsumerRequestV0) { //make competibel
```
Comment (shay23b) [inline server/memphis_handlers_consumers.go:594]: competibel -> compatible

Comment (daniel-davidd) [reply server/memphis_handlers_consumers.go:594]: deleted the comment, it was for me

**Review by shay23b — APPROVED** [review]

**Pull request merged**